Tests/UserDefaultsDependencyTests/UserDefaultsDependencyTests.swift

```diff
@@ -7,144 +7,153 @@ final class UserDefaultsDependencyTests: XCTestCase {
   func testEphemeralDefaultsDate() {
     let ephemeral = UserDefaults.Dependency.ephemeral()
     let date = Date(timeIntervalSince1970: 1000)
+    let key = #function
     withDependencies {
       $0.userDefaults = ephemeral
     } operation: {
-      self.userDefaults.set(date, forKey: "date")
+      self.userDefaults.set(date, forKey: key)
     }
     withDependencies {
       $0.userDefaults = ephemeral
     } operation: {
-      XCTAssertEqual(self.userDefaults.date(forKey: "date"), date)
+      XCTAssertEqual(self.userDefaults.date(forKey: key), date)
     }
   }
 
   func testLiveUserDefaultsBool() {
     let bool = true
-    UserDefaults.standard.removeObject(forKey: "bool")
+    let key = #function
+    UserDefaults.standard.removeObject(forKey: key)
     withDependencies {
       $0.userDefaults = .standard
     } operation: {
-      self.userDefaults.set(bool, forKey: "bool")
+      self.userDefaults.set(bool, forKey: key)
     }
     withDependencies {
       $0.userDefaults = .standard
     } operation: {
-      XCTAssertEqual(self.userDefaults.bool(forKey: "bool"), bool)
+      XCTAssertEqual(self.userDefaults.bool(forKey: key), bool)
     }
-    UserDefaults.standard.removeObject(forKey: "double")
```

Comment:
Originally I was just gonna fix this inaccuarcy, where key `"double"`, but where `"bool"` was intended, then I figured why not use `#function` as key?